### DISCUSSION TITLE
Sync

### DISCUSSION THREAD

**Discussion by JustinElrod** [dudor/BookmarkHub#45] [General]
Hi. I am trying to merge my bookmarks from chrome and edge but I'm not sure I am using the tool right as it only pops up a notification that the bookmarks across browsers are different. The problem I am having is that I use edge and chrome a lot and daily both browsers accumulate different bookmarks and seems like this tool only transfers all bookmarks from one browser to another and doesn't have an option to merge.

**Comment by yannisdran** [comment]
There is no sync tool, only a "download" and an "upload" button. This is bad, because sometimes it is difficult to remember which browser has the latest version and which one tries to add a new branch to the main uploaded version.

**Comment by newbie992** [comment]
I am not able to create a git-ID.  The git-token is created, but from there the installation tutorial is not understandable for me.
No chance for newbies, only for computer-experts.
